interface/main/finder/dynamic_finder.php

```diff
@@ -337,48 +341,41 @@ function persistCriteria(el, e) {
     $oemr_ui = new OemrUI($arrOeUiSettings);
     ?>
 </head>
-<body class="body_top">
-    <div id="container_div" class="<?php echo attr($oemr_ui->oeContainer()); ?>">
+<body>
+    <div id="container_div" class="<?php echo attr($oemr_ui->oeContainer()); ?> mt-3">
          <div class="row">
-            <div class="col-sm-12">
-                <div class="page-header">
-                    <?php echo $oemr_ui->pageHeading() . "\r\n"; ?>
-                </div>
+            <div class="col-12">
```

Comment:
Remove this col-12 and row div and replace with w-100

Reply:
Do you mean all of `col-12` class? If not, why we need to remove all of the `row` class, Is it not break the grid system?

Reply:
The col-12 really does no harm in terms of coding yet it is a legacy term that shouldn't really be used unless you are making multiple col's. In this instance the row class only has one child col class, so both the row and the col can be removed.